books/RayTracingInOneWeekend.html

```diff
@@ -1057,6 +1060,9 @@
         bool front_face;
 
         void set_face_normal(const ray& r, const vec3& outward_normal) {
+            // Sets the hit record normal vector.
+            // NOTE: the parameter `outward_normal` must be of unit length.
```

Comment:
Replace "must" with "is assumed to"

There is no code enforcement, a reader might read this and assume from MUST that there is some code of enforcement (e.g. their code won't compile, it'll crash).

We have no guarantees that a non-unit length normal will be problematic, and so I don't think MUST is appropriate